torch/_utils_internal.py

```diff
@@ -33,5 +34,23 @@ def resolve_library_path(path):
     return os.path.realpath(path)
 
 
+def get_source_lines_and_file(obj):
```

Comment:
This should probably live in `_jit_internal.py` instead

Reply:
_utils_internal.py is a magical file that can be overloaded in the build system with better error messages or internal-specific tweaks (see the comment). So that's why it lives in _utils_internal.